ols/app/models/config.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, data: Optional[dict] = None):
         """Initialize configuration and perform basic validation."""
         super().__init__()
         if data is None:
-            return
+            data = {}
```

Comment:
`data = data or {}`?

Reply:
actually I was thinking that later we might inform about that state (dunno why when logger is not setup at this stage ;)

Reply:
WDYT @onmete ?

Reply:
ok